sycl/doc/EnvironmentVariables.md

```diff
@@ -30,6 +30,7 @@ subject to change. Do not rely on these variables in production code.
 | SYCL_PI_LEVEL_ZERO_DISABLE_USM_ALLOCATOR | Any(\*) | Disable USM allocator in Level Zero plugin (each memory request will go directly to Level Zero runtime) |
 | SYCL_PI_LEVEL_ZERO_BATCH_SIZE | Integer | Sets a preferred number of commands to batch into a command list before executing the command list. A value of 0 causes the batch size to be adjusted dynamically. A value greater than 0 specifies fixed size batching, with the batch size set to the specified value. The default is 0. |
 | SYCL_PI_LEVEL_ZERO_FILTER_EVENT_WAIT_LIST | Integer | When set to 0, disables filtering of signaled events from wait lists when using the Level Zero backend. The default is 1. |
+| SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE | Integer | Enable use of copy engine, if available in the device, in Level Zero plugin to transfer SYCL buffer or image data between the host and/or device(s). The default is 1. |
```

Comment:
1) and also for "fill" operations
2) I think we should explicitly that this doesn't force blitter, but just allows. final decision is by heuristics. and then maybe we should add force semantics too?

Reply:
I modified the comment to include 'fill' operations. Since the heuristics controlling the blitter usage are beyond the plugin scope, I am not sure if we need to add anything here.

Reply:
Modified text. Since heuristics is not part of plugin code, I suggest we do not add any 'force' semantics here.